site/content/en/docs/handbook/addons/headlamp.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Using Headlamp Addon"
+linkTitle: "Headlamp"
+weight: 1
+date: 2022-06-08
+---
+
+
+## Headlamp Addon
+[Headlamp](https://github.com/kinvolk/headlamp) - Headlamp is an easy-to-use and extensible Kubernetes web UI.
```

Comment:
check the comments on the README, which should apply here too.

Reply:
Removed the README and added the link to the docs in the addon description.